pkg/controller/controller_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
why did we name it `dto`? Should be named something else.

Reply:
`dto` was borrowed from the prometheus unit test examples. I've renamed this to `promtest`. Let me know if it works.